hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestInterfaceAlign.java

```diff
@@ -56,6 +56,7 @@ public void testAdminWithAsyncAdmin() {
     adminMethodNames.remove("getConfiguration");
     adminMethodNames.removeAll(getMethodNames(Abortable.class));
     adminMethodNames.removeAll(getMethodNames(Closeable.class));
+    asyncAdminMethodNames.remove("coprocessorServiceOnAllRegionServers");
```

Comment:
same as master branch PR, we might not need this change anymore

Reply:
+1.